### PR TITLE
Issue#690: fixed Attendance Button

### PR DESCRIPTION
**Pull request by raluka**

### CODE REVIEW

**Comment by differentreality** [comment]
Do you think it would make sense to keep the notice/alert messages in the controller?

File: app/controllers/admin/registrations_controller.rb [hunk 1]
```diff
@@ -11,7 +11,8 @@ def index
       @attended = @conference.registrations.where('attended = ?', true).count
     end
 
-    def edit; end
+    def edit;
```
Comment (ChrisBr) [inline app/controllers/admin/registrations_controller.rb]: Please keep the old syntax 

```
```

**Comment by ChrisBr** [comment]
I really like the refactoring of the absent & present actions :+1:

**Comment by ChrisBr** [comment]
@differentreality Hm, yes a visual notification would definitely makes sense ... But as we now use AJAX I'm not sure if we can keep it in the controller ...

**Comment by ChrisBr** [comment]
Otherwise LGTM

**Comment by differentreality** [comment]
@ChrisBr  what about sth like https://github.com/leonid-shevtsov/unobtrusive_flash ?

**Comment by hennevogel** [comment]
The button changes. What other notification do you need? :-)

**Comment by differentreality** [comment]
@hennevogel I was thinking flash message, so that we know which user/registration we affected, since this is in the index page and in case of wrong user it could be proven very helpful to see it on the message.

**Comment by ChrisBr** [comment]
@hennevogel yes, sure! But if sth goes wrong in the backend the button will change too ...
But I agree, we shouldn't Overengineer this ... we can add the notification in a later PR!

File: app/views/admin/registrations/index.html.haml [hunk 2]
```diff
@@ -44,24 +44,15 @@
               %td
                 = link_to 'Questions','#', class: 'btn btn-success question-btn', 'data-id' => index, 'data-name' => registration.name
             %td
+              = check_box_tag "#{@conference.short_title}_#{registration.id}", registration.id, registration.attended,
+                class: 'switch-checkbox', method: :patch,
+                url: toggle_attendance_admin_conference_registration_path(@conference.short_title, id: registration.id)+"?attended=",
+                data: { size: "large",
```
Comment (ChrisBr) [inline app/views/admin/registrations/index.html.haml]: You should use size: 'small' like in #693 discussed ... Large still looks a little bit ugly :(

**Comment by ChrisBr** [comment]
Okay, when you finally fix the size and squash your commits together we're ready to :ship: 

Good work :clap: :clap: :clap:

File: app/views/admin/registrations/index.html.haml (hunk 2 above)
Comment (raluka) [inline app/views/admin/registrations/index.html.haml]: ok, will do it

**Comment by differentreality** [comment]
@ChrisBr even with 'small' the button doesnt look very good... 

screenshot

**Comment by raluka** [comment]
Large button vs small button. Any option need a bit of CSS to make it acceptable.
large_button
small_button

**Comment by hennevogel** [comment]
Changing the appearance of the bootstrap-switch buttons is not the scope of this pull request.

**Pull request merged**